<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LinguistMerge</string>
  </property>
  <widget class="QWidget" name="m_centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeView" name="m_treeView">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ContiguousSelection</enum>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="m_toolBar">
   <property name="windowTitle">
    <string>Merging</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_actionAdd"/>
   <addaction name="m_actionRemove"/>
   <addaction name="m_actionClear"/>
   <addaction name="separator"/>
   <addaction name="m_actionMerge"/>
  </widget>
  <widget class="QMenuBar" name="m_menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="m_actionExit"/>
   </widget>
   <widget class="QMenu" name="m_menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="m_actionAbout"/>
   </widget>
   <widget class="QMenu" name="m_menuMerging">
    <property name="title">
     <string>&amp;Merging</string>
    </property>
    <addaction name="m_actionAdd"/>
    <addaction name="m_actionRemove"/>
    <addaction name="m_actionClear"/>
    <addaction name="separator"/>
    <addaction name="m_actionMerge"/>
   </widget>
   <addaction name="m_menuFile"/>
   <addaction name="m_menuMerging"/>
   <addaction name="m_menuHelp"/>
  </widget>
  <action name="m_actionAdd">
   <property name="icon">
    <iconset resource="linguistmerge.qrc">
     <normaloff>:/LinguistMerge/icons/add.svg</normaloff>:/LinguistMerge/icons/add.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Add</string>
   </property>
  </action>
  <action name="m_actionMerge">
   <property name="icon">
    <iconset resource="linguistmerge.qrc">
     <normaloff>:/LinguistMerge/icons/merge.svg</normaloff>:/LinguistMerge/icons/merge.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Merge</string>
   </property>
  </action>
  <action name="m_actionRemove">
   <property name="icon">
    <iconset resource="linguistmerge.qrc">
     <normaloff>:/LinguistMerge/icons/remove.svg</normaloff>:/LinguistMerge/icons/remove.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Remove</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="m_actionClear">
   <property name="icon">
    <iconset resource="linguistmerge.qrc">
     <normaloff>:/LinguistMerge/icons/clear.svg</normaloff>:/LinguistMerge/icons/clear.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </action>
  <action name="m_actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="m_actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionContents">
   <property name="text">
    <string>Contents</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>m_treeView</tabstop>
 </tabstops>
 <resources>
  <include location="linguistmerge.qrc"/>
 </resources>
 <connections/>
</ui>
